interfaces/portal/src/app/services/get-registration-preferrred-language-name.service.ts

```diff
@@ -0,0 +1,69 @@
+import { inject, Injectable, LOCALE_ID } from '@angular/core';
+
+import { RegistrationPreferredLanguage } from '@121-service/src/shared/enum/registration-preferred-language.enum';
+import { UILanguage } from '@121-service/src/shared/enum/ui-language.enum';
+
+import { languageCodesToLinguonymsForArab } from '~/services/linguonyms/arab';
+import { languageCodesToLinguonymsForDutch } from '~/services/linguonyms/dutch';
+import { languageCodesToLinguonymsForEnglish } from '~/services/linguonyms/english';
+import { languageCodesToLinguonymsForFrench } from '~/services/linguonyms/french';
+import { languageCodesToLinguonymsForSlovak } from '~/services/linguonyms/slovak';
+import { languageCodesToLinguonymsForSpanish } from '~/services/linguonyms/spanish';
```

Comment:
I guess this answers my previous question in part (about something breaking). I guess my question is my general: is it clear enough what to do upon adding a new ui language? Is there documentation also?